src/main/kotlin/net/atos/zac/smartdocuments/SmartDocumentsService.kt

```diff
@@ -46,47 +46,65 @@ class SmartDocumentsService @Inject constructor(
     fun listTemplates() = documentCreatieService.listTemplates().toREST()
 
     /**
-     * Stores template mapping for ZaakafhandelParameters
+     * Stores template mapping for ZaakafhandelParameters and InformatieObjectType
```

Comment:
In JavaDoc I would not use actual class names in text because they will always go out of sync with refactoring. If we do we should use actual JavaDoc links so that they are up to date.
```suggestion
     * Stores template mapping for zaakafhandelparameters and informatieobjecttypes.
```